src/components/admin/article/ArticleController.jsx

```diff
@@ -60,70 +64,70 @@ class ArticleController extends FalcorController {
     }, 500);
   }
 
-  save(jsonGraphEnvelope, processedAuthors, articleSlug, falcorData) {
+  async save(jsonGraphEnvelope, processedStaff, articleId, falcorData) {
     // Update the values
     this.safeSetState({ saving: true });
-    const updatePromises = [this.falcorUpdate(jsonGraphEnvelope)];
+    const dbUpdates = [this.falcorUpdate(jsonGraphEnvelope)];
 
-    if (processedAuthors !== null) {
-      updatePromises.push(
+    if (processedStaff !== null) {
+      dbUpdates.push(
         this.falcorCall(
-          ['articles', 'bySlug', articleSlug, 'authors', 'updateAuthors'],
-          [falcorData.id, processedAuthors],
+          ['articles', 'byId', articleId, 'staff', 'updateStaff'],
```

Comment:
I actually purposefully left the authors here, since for articles it's actually authors, I guess if you find it confusing we could maybe not do that though? I left authors in every place where it was in relation to being an author of an article. This is especially because in the future we plan to add in "media contributors", like illustrators, videomakers etc. so at that point it's good to have the distinction, so yeah, as I'm writing I think I don't want you to change this.

Reply:
Hmm, then you must have missed changing this https://github.com/thegazelle-ad/gazelle-server/pull/345/files#diff-3be8563bccafa48d33411bd23dffdb58R171 
You had two different paths, one Authors and one Staff all under articles. Having an 'authors' route that updates the 'staff' path seems not very Falcor, if I am understanding correctly.

Reply:
@emilgoldsmith Just to make sure, am I correct that the remaining staff there were missed in F&R? They were causing bugs for me, but I want to make sure that they weren't somehow intended

Reply:
Oh, I guess that link doesn't take you to the highlighted line like I thought it did.

Reply:
Oh right I was gonna talk to you about that, and yeah it doesn't take me to any line I'm afraid, let's take it on Slack as it's better for instant messaging